### PR TITLE
[platform API][masic] Skip platform API tests on multi ASIC platforms running 201911

### LINKED ISSUE #5159: [masic][pytest fails] Platform API tests are failing on multi ASIC 201911 image
**Description**

        physical_port_index_map = get_physical_port_indices(duthost, physical_intfs)

        sfp_port_indices = set([physical_port_index_map[intf] \
            for intf in physical_port_index_map.keys()])
        sfp_setup["sfp_port_indices"] = sorted(sfp_port_indices)

        if len(xcvr_skip_list[duthost.hostname]):
            logging.info("Skipping tests on {}".format(xcvr_skip_list[duthost.hostname]))

        sfp_port_indices = set([physical_port_index_map[intf] for intf in \
                                                    physical_port_index_map.keys() \
                                                    if intf not in xcvr_skip_list[duthost.hostname]])
        sfp_setup["sfp_test_port_indices"] = sorted(sfp_port_indices)

        # Fetch SFP names from platform.json
        sfp_fact_names = []
>       sfp_fact_list = duthost.facts.get("chassis").get("sfps")
E       AttributeError: 'NoneType' object has no attribute 'get'

platform_tests/api/test_sfp.py:59: AttributeError

**Steps to reproduce the issue:**
1.
2.
3.

**Describe the results you received:**

**Describe the results you expected:**

**Additional information you deem important:**

    **Output of `show version`:**

    ```
    (paste your output here)
    ```

    **Attach debug file `sudo generate_dump`:**

    ```
    (paste your output here)
    ```

### PR DESCRIPTION
**Pull request by smaheshm**
### Description of PR

Summary:
Fixes #5159

### Type of change

- [ ] Bug fix
- [x ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 201911

#### What is the motivation for this PR?
New platform API tests are not yet supported on multi ASIC platform running 201911 image. 

#### How did you do it?
Skip tests on multi ASIC platfrom running 201911 release.

#### How did you verify/test it?
```
samaddik@svcstr-server-2:/var/sonic-mgmt/tests$ pytest platform_tests/api  --testbed=vmsvc1-t1-nmasic-acs-1 --inventory=../ansible/strsvc,../ansible/veos  --testbed_file=../ansible/testbed.csv --host-pattern=svcstr-nmasic-acs-1  --module-path=../ansible/library --skip_sanity
/usr/local/lib/python2.7/dist-packages/ansible/parsing/vault/__init__.py:44: CryptographyDeprecationWarning: Python 2 is no longer supported by the Python core team. Support for it is now deprecated in cryptography, and will be removed in the next release.
  from cryptography.exceptions import InvalidSignature
============================================================================================================ test session starts ============================================================================================================
platform linux2 -- Python 2.7.17, pytest-4.6.5, py-1.10.0, pluggy-0.13.1
ansible: 2.8.12
rootdir: /var/sonic-mgmt/tests, inifile: pytest.ini
plugins: html-1.22.1, repeat-0.9.1, forked-1.3.0, xdist-1.28.0, metadata-1.11.0, ansible-2.2.2
collecting ... ['conf-name', 'group-name', 'topo', 'ptf_image_name', 'ptf', 'ptf_ip', 'ptf_ipv6', 'server', 'vm_base', 'dut', 'inv_name', 'auto_recover', 'comment']
Finished testbed info generating.
/usr/local/lib/python2.7/dist-packages/ansible/parsing/vault/__init__.py:44: CryptographyDeprecationWarning: Python 2 is no longer supported by the Python core team. Support for it is now deprecated in cryptography, and will be removed in the next release.
  from cryptography.exceptions import InvalidSignature
collected 169 items

platform_tests/api/test_chassis.py ssssssssssssssssssssssss                                                                                                                                                                           [ 14%]
platform_tests/api/test_chassis_fans.py sssssssssssss                                                                                                                                                                                 [ 21%]
platform_tests/api/test_component.py sssssssssssss                                                                                                                                                                                    [ 29%]
platform_tests/api/test_fan_drawer.py sssssssssss                                                                                                                                                                                     [ 36%]
platform_tests/api/test_fan_drawer_fans.py sssssssssssss                                                                                                                                                                              [ 43%]
platform_tests/api/test_module.py ssssssssssssssssssssss                                                                                                                                                                              [ 56%]
platform_tests/api/test_psu.py ssssssssssssss                                                                                                                                                                                         [ 65%]
platform_tests/api/test_psu_fans.py sssssssssssss                                                                                                                                                                                     [ 72%]
platform_tests/api/test_sfp.py sssssssssssssssssssssss                                                                                                                                                                                [ 86%]
platform_tests/api/test_thermal.py ssssssssssssssss                                                                                                                                                                                   [ 95%]
platform_tests/api/test_watchdog.py sssssss                                                                                                                                                                                           [100%]

============================================================================================================= warnings summary ==============================================================================================================
/usr/local/lib/python2.7/dist-packages/_pytest/config/__init__.py:538
  /usr/local/lib/python2.7/dist-packages/_pytest/config/__init__.py:538: PytestAssertRewriteWarning: Module already imported so cannot be rewritten: tests.common.dualtor
    self.import_plugin(import_spec)

/usr/local/lib/python2.7/dist-packages/_pytest/mark/structures.py:334
  /usr/local/lib/python2.7/dist-packages/_pytest/mark/structures.py:334: PytestUnknownMarkWarning: Unknown pytest.mark.dependency - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/latest/mark.html
    PytestUnknownMarkWarning,

-- Docs: https://docs.pytest.org/en/latest/warnings.html
================================================================================================== 169 skipped, 2 warnings in 5.09 seconds ==================================================================================================
s
```

### CODE REVIEW

**Comment by prgeor** [comment]
@smaheshm could you provide the issue link in the description?

File: tests/common/plugins/conditional_mark/tests_mark_conditions_platform_tests.yaml [hunk 1]
```diff
@@ -1,6 +1,12 @@
 #######################################
 #####      api/test_chassis.py    #####
 #######################################
+platform_tests/api:
+  skip:
+    reason: "Unsupported platform API"
+    conditions:
+      - "is_multi_asic==True and release in ['201911']"
+
```
Comment (prgeor) [inline tests/common/plugins/conditional_mark/tests_mark_conditions_platform_tests.yaml:9]: could you put this block at the start of the file

**Review by prgeor — DISMISSED** [review]

**Review by prgeor — APPROVED** [review]

**Review by judyjoseph — APPROVED** [review]

**Pull request merged**